region/maridia/inner-pink.json

```diff
@@ -7451,16 +7489,7 @@
                     "canSuitlessMaridia",
                     "canSpringBallJumpMidAir",
                     {"or": [
-                      "HiJump",
                       "canCarefulJump",
-                      {"and": [
```

Comment:
Why remove all of these?

Reply:
the HiJump one was already listed in the strat above, and you can get out of the water with just a spring ball jump .... But now I'm testing it more, I think its a bit harder than should be expected with just a normal spring ball jump, so I'll revert some of these.